design/overview.md

```diff
@@ -29,7 +29,8 @@ Key concepts that we mention below will be defined here:
 * `Operator` - This is a person (or people) who control a Cosmos SDK validator node. This is also called `valoper` or "Validator Operator" in the Cosmos SDK staking section
 * `Full Node` - This is an *Ethereum* Full Node run by an Operator
 * `Validator` - This is a Cosmos SDK Validating Node (signing blocks)
-* `Orchestrator` (name WIP) - This is a Rust binary controlled by an Operator that holds Cosmos SDK and Ethereum private keys used for signing transactions used to move tokens between the two chains. (This was called peggy module in other docs, which is confusing as the module runs on the node... I would love a better name than Orchestrator)
+* `Orchestrator` (name WIP) - This is a Rust binary controlled by an Operator that holds Cosmos SDK and Ethereum private keys used for signing transactions used to move tokens between the two chains. It's important to keep in mind that the `Orchestrator` binary exists only as an implementation detail, to help the `Peggy module` interface with Ethereum. In general, if something has been decided on by the `Peggy module` and a `Validator`'s `Orchestrator` does not sign it quickly enough, this is considered downtime, since the `Validator` already agreed to it in principle through the consensus process.
```

Comment:
this isn't very clear in my opinion, can we break out the concept of what is and isn't downtime separately? I feel like it's too much to address as an aside here.

Reply:
Yea, i can see that viewpoint. The reason I call it out is to reinforce that the Orchestrators are not independent entities, but simply an artifact of code organization. I think the word "Orchestrator" kind of implies that they are. Honestly, I would have chosen a different term but I don't want to get us hung up on terminology bikeshedding.

Reply:
my issue is that we're saying it will be slashed but not outlining how or making a TODO for it. Which is part of what we're trying to avoid with a detailed spec. It should make clear what is designed and isn't designed yet.

Reply:
I think that if we get overly preoccupied with trying to write a full IETF style spec we will end up spending a huge amount of time on it like you were worried about. I'm trying to get down a sense of the general structure and goals. Let's come back revise this document once all the slashing is thought out.

Reply:
Let's keep slashing out of this for now. I have added issue #42 to discuss this later